symphony/app/fbcnms-projects/magmalte/app/theme/default.js

```diff
@@ -230,6 +230,13 @@ export default createMuiTheme({
       contained: {
         boxShadow: 'none',
       },
+      containedPrimary: {
```

Comment:
where does this get used ? is this a replacement for appBarBtn, appBarBtnSecondary ?

Reply:
This is the start of some default stylings for the content buttons we use across the site. An example of this being used can be found in **Equipment > Gateway Detail View > Logs** page, with the **Export** button.

![image](https://user-images.githubusercontent.com/8878152/87310537-1614a880-c4ec-11ea-8e90-c956ad1f452f.png)